### PR TITLE
Introduces a test for PR #818 and issue #817 local sdists

### PR DESCRIPTION
**Pull request by techalchemy**
Signed-off-by: Dan Ryan <dan@danryan.co>

### CODE REVIEW

File: tests/test_pipenv.py [hunk 1]
```diff
@@ -651,3 +652,26 @@ def test_urls_work(self):
             dep = p.lockfile['default'][key]
 
             assert 'file' in dep
+
+    @pytest.mark.install
+    @pytest.mark.files
+    def test_local_zipfiles(self):
+
+        with PipenvInstance() as p:
+            zip_contents = requests.get('https://github.com/divio/django-cms/archive/release/3.4.x.zip')
```
Comment (nateprewitt) [inline tests/test_pipenv.py]: I'd prefer to either find a way to stub this out or bundle a condensed example.zip package if we can. Network calls tend to be where a lot of test time and failures come from.

Comment (techalchemy) [reply tests/test_pipenv.py]: Sure I wasn't totally sure about the best practice for this so I just copied the test above it (though to be fair that is testing a URL as well).  I didn't want to just dump this in the repo without talking ot someone though.  Might want to find something smaller to be honest.  Where should I put it?

Comment (nateprewitt) [reply tests/test_pipenv.py]: Sorry I missed this last night @techalchemy. I'd create a directory under tests with "test_artifacts" are something like that and drop it in there.

**Comment by techalchemy** [comment]
OK @nateprewitt sorry for being slow but I got around to making those changes.  I hope this doesn't break travis/appveyor, but we shall see

**Comment by nateprewitt** [comment]
No problem, thanks for getting things put together :) I won't have a chance to look at this till I get home but if we get green builds, I think we'll be set to merge.

**Comment by techalchemy** [comment]
@nateprewitt the zip I used was grabbed from tag 0.12.1 on github a few hours ago with this command exactly:

```console
$ wget -O tablib.zip https://github.com/kennethreitz/tablib/archive/v0.12.1.zip
```

I went ahead and grabbed the tar.gz from pypi however, and since this test is more about ensuring that installing an archive in the current directory works (i.e. with the command `pipenv install tablib.zip`), the same test is valid using a tar.gz instead so I just grabbed the one you linked and dropped it in as a replacement with no modification.

**Comment by nateprewitt** [comment]
@techalchemy, ah ok, there must be something on master that varies from the last release which is likely why things weren't lining up. Thanks for taking care of that so quick. Once the Appveyor wraps, I'll merge.

**Comment by nateprewitt** [comment]
:sparkles:

**Pull request merged**